server/parser/src/main/antlr4/org/eclipse/lsp/cobol/core/parser/CobolParser.g4

```diff
@@ -417,7 +417,7 @@ fileDescriptionEntry
    ;
 
 fileDescriptionEntryClauses
-   : (FD | SD) cobolWord (DOT_FS? fileDescriptionEntryClause)* dot_fs   ;
```

Comment:
Could you please also update https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/blob/0094f340a262fcf1d4aaa53b88b205116e0ce12c/server/parser/src/main/antlr4/org/eclipse/lsp/cobol/core/parser/CobolDataDivisionParser.g4 this one